pkg/apis/core/validation/validation.go

```diff
@@ -2962,10 +2962,16 @@ const (
 	// restrictions in Linux libc name resolution handling.
 	// Max number of DNS name servers.
 	MaxDNSNameservers = 3
-	// Max number of domains in search path.
-	MaxDNSSearchPaths = 6
-	// Max number of characters in search path.
-	MaxDNSSearchListChars = 256
+	// Max number of domains in the search path list.
+	MaxDNSSearchPaths = 32
```

Comment:
suggest picking new names (MaxDNSSearchPathsExpanded, MaxDNSSearchPathsLegacy) rather than changing constant values in ways that can silently break existing consumers